src/main/scala/com/microsoft/hyperspace/index/sources/default/DefaultFileBasedSource.scala

```diff
@@ -201,9 +214,9 @@ class DefaultFileBasedSource(private val spark: SparkSession) extends FileBasedS
    * @param location Partitioned data location.
    * @return basePath to read the given partitioned location.
```

Comment:
Can you update the comment what `Some(None)` will mean? (also in the `interfaces.scala`)

Reply:
or you can just say `optional basePath`.

Reply:
thanks @imback82 , done